docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md

```diff
@@ -39,7 +39,73 @@ Make sure that the following modules have been installed:
 
 {% endinfo_block %}
 
-### 2) Set up transfer objects
+### 2) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN | DESCRIPTION | PREREQUISITES | NAMESPACE |
+|-|-|-|-|
+| MerchantProductProductAbstractPostCreatePlugin | Creates a new merchant product abstract entity if `ProductAbstractTransfer.idMerchant` is set. | None | Spryker\Zed\MerchantProduct\Communication\Plugin\Product |
```

Comment:
Product is a module of `Merchant product feature`, please move to `Merchant product feature` IG